packages/service-clients/odsp-client/api-report/odsp-client.api.md

```diff
@@ -61,6 +61,8 @@ export interface OdspMember extends IMember {
     email: string;
     // (undocumented)
     name: string;
+    // (undocumented)
+    oid: string;
```

Comment:
isn't there already an id field? what is in that?

Reply:
id and email are same in the response.

Reply:
oid is a guid

Reply:
do we need two emails? should oid be in the userId field? the current structure is confusing and redundant.